src/microsoft/office/license/user/microsoft-office-license-user.controller.js

```diff
@@ -35,25 +35,17 @@ angular.module("Module.microsoft.controllers").controller("MicrosoftOfficeLicens
                     this.loaders.userPoll[item] = true;
```

Comment:
I don't think you use that variable anymore.

Reply:
yes. passing item.isLoading is a better way to do it.
Done